src/com/model/BanknotesCounter.java

```diff
@@ -0,0 +1,31 @@
+package com.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BanknotesCounter {
+    // Allowed banknotes to be deposited;
+    public final int[] ALLOWED_BANKNOTES = {1, 3, 5, 10, 20, 100};
+    // The key is the denomination of the banknote. Value - its quantity in stock;
+    public Map<Integer, Integer> banknotesMap = new HashMap<>();
```

Comment:
(cosmetic) You would be right to declare this field as `final` because a reference to this field will never change (i.e., there is nothing like `banknotesMap = someOtherMap` nowhere in the code